pom.xml

```diff
@@ -106,11 +119,21 @@
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>branch-api</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.jenkins-ci.plugins.workflow</groupId>
+      <artifactId>workflow-multibranch</artifactId>
+      <version>2.21</version>
```

Comment:
I suggest to drop the version from here to let the Jenkins BOM plugin do its work.

Reply:
Forget about this, it seems it is not managed by the Jenkins BOM plugin.